tensorflow/python/layers/normalization.py

```diff
@@ -248,6 +263,7 @@ def build(self, input_shape):
         def _renorm_variable(name, shape):
           var = self.add_variable(name=name,
                                   shape=shape,
+                                  dtype=param_dtype,
                                   initializer=init_ops.zeros_initializer(),
                                   trainable=False)
           return var
```

Comment:
In the function "batch_normalization" below, when calling "BatchNormalization", pass the dtype of the input tensor.

Reply:
Done